### ISSUE TITLE
About the new model mobilefacenet of InsightFace

### ISSUE THREAD

**Issue by HsuTzuJen** [auroua/InsightFace_TF#23]
Hi, I just modify the code for the new model mobile-facenet of InsightFace"https://github.com/deepinsight/insightface/blob/master/src/symbols/fmobilefacenet.py"
to TF version as follow:
"https://github.com/HsuTzuJen/Face_Recognition_Practice_with_TF/blob/master/Nets/mobilefacenet_test.py"
I set the training parameters as the original paper"https://arxiv.org/abs/1804.07573", but it is not converged correctly.
Can you please help me to find out what is going on?

**Comment by ruobop** [comment]
Hi, @HsuTzuJen

**Comment by billtiger** [comment]
I also encounter this problem about when lr>0.05,after a moment the total loss is nan,so i use the learning rate start at 0.02,i don't know why?can you solve it?
At the moment, the best accuracy i can achieve is Accuracy-Flip: 0.98150+-0.00545.

**Comment by HsuTzuJen** [comment]
@billtiger The best acc I achieved is 0.9875 with batch size 64 at step 348000.

**Comment by billtiger** [comment]
@HsuTzuJen thank you for your reply!
but I think the learning rate can't be set at 0.1 is abnormal,and this will result low accuracy!

**Comment by billtiger** [comment]
@HsuTzuJen how about the prelu and leaky_relu influence the accuracy?
i find the the original paper authors are use PReLU,and you and me are all used leaky_relu,
and i don't find PReLU api in tensorflow tf.nn module.so i use the tf.nn.leaky_relu,
have you ever try PReLU?hope you reply!thanks!

**Comment by HsuTzuJen** [comment]
@billtiger As I know, prelu is leaky relu.

**Comment by HsuTzuJen** [comment]
@billtiger You can try:
    bn = tl.layers.InputLayer(bn)
    act = tl.layers.PReluLayer(bn, name='%s%s_Prelu' %(name, suffix))
    
    return act.outputs

Leaky_relu is a Prelu with a stable alpha(not trainable), but the alpha is trainable in Prelu.

**Comment by siahewei** [comment]
why learning rate is so small as train.py set 0.01, something wrong?

**Comment by lfb94** [comment]
Hello,

Any progress? Do you know where I can find the pretrained model for mobilefacenet?

Thanks!

**Comment by Drsea123** [comment]
Could you tell me about your model's learning rate and How much steps you can arrival the 99+ accuracy for your model? please! @HsuTzuJen

**Comment by 406747925** [comment]
@HsuTzuJen  do you use weight decay for mobilenet? and what lr step did you use ,thx

**Comment by HsuTzuJen** [comment]
@406747925 Please check on "https://github.com/HsuTzuJen/Face_Recognition_Practice_with_TF"
I have uploaded the codes that I am using.
